openapi.json

```
@@ -1895,7 +2147,7 @@
         }
       },
       "post": {
-        "summary": "Publish a new secondary web service.",
+        "summary": "Publish a new service",
         "description": "Calling this endpoint will create a secondary web service such as WMTS, TMS or WCS. The underlying data is processes on-demand, but a process graph may simply access results from a batch job. Computations should be performed in the sense that it is only evaluated for the requested spatial / temporal extent and resolution.\n\n**Note:** Costs incured by shared secondary web services are usually paid by the owner, but this depends on the service type and whether it supports charging fees or not.",
```

Costs incured --> Costs incurred

```suggestion
        "description": "Calling this endpoint will create a secondary web service such as WMTS, TMS or WCS. The underlying data is processes on-demand, but a process graph may simply access results from a batch job. Computations should be performed in the sense that it is only evaluated for the requested spatial / temporal extent and resolution.\n\n**Note:** Costs incurred by shared secondary web services are usually paid by the owner, but this depends on the service type and whether it supports charging fees or not.",
```

---

I only found this minor issue (misspelling). Other than that, great job @m-mohr !

---

Thanks.